acceptance/acceptance_test.go

```diff
@@ -220,17 +220,39 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 	}
 
 	cloudEnv := os.Getenv("CLOUD_ENV")
-	if !isTruePtr(config.Local) && cloudEnv == "" {
-		t.Skipf("Disabled via Local setting in %s (CLOUD_ENV=%s)", configPath, cloudEnv)
-	}
+	isRunningOnCloud := cloudEnv != ""
+	tailOutput := Tail
 
-	if !isTruePtr(config.Cloud) && cloudEnv != "" {
-		t.Skipf("Disabled via Cloud setting in %s (CLOUD_ENV=%s)", configPath, cloudEnv)
-	}
+	if isRunningOnCloud {
+		if isTruePtr(config.CloudLong) {
+			if testing.Short() {
+				t.Skipf("Disabled via CloudLong setting in %s (CLOUD_ENV=%s, Short=%v)", configPath, cloudEnv, testing.Short())
+			}
+
+			if testing.Verbose() {
+				// Combination of CloudLong and -v auto-enables -tail
+				tailOutput = true
```

Comment:
Unrelated, but tailing the output can be the default behaviour for all tests.

Reply:
I don't think it's needed, since we already record it and show a diff, it's only useful to monitor and review timestamps of long running tests.